bcda/cclf/cclf.go

```diff
@@ -179,15 +168,13 @@ func importCCLF(ctx context.Context, fileMetadata *cclfFileMetadata, importer im
 		fmt.Printf("Could not read CCLF%d archive %s.\n", fileMetadata.cclfNum, fileMetadata.filePath)
 		err := errors.Wrapf(err, "could not read CCLF%d archive %s", fileMetadata.cclfNum, fileMetadata.filePath)
```

Comment:
This would be another instance where we want to switch to `err = errors.Wrapf(err, "could not read CCLF%d archive %s", fileMetadata.cclfNum, fileMetadata.filePath)`